test/lit/wat-kitchen-sink.wast

```diff
@@ -121,6 +121,66 @@
  (func $f4 (type 13) (local i32 i64) (local $l f32))
  (func (export "f5.0") (export "f5.1") (import "mod" "f5"))
 
+ ;; CHECK:      (func $nop-skate (type $void)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT: )
+ (func $nop-skate
+  nop
+  nop
+  nop
+  nop
+  nop
+ )
+
+ ;; CHECK:      (func $nop-ski (type $void)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT:  (nop)
+ ;; CHECK-NEXT: )
+ (func $nop-ski
+  (nop
+   (nop
+    (nop)
+    (nop)
+    (nop
+     (nop)
+    )
+   )
+   (nop)
+  )
+  (nop)
+ )
```

Comment:
Rather than nops perhaps these could be drops of constants, which would make it possible to see the order is correct?

Reply:
I'll have to drop my funny (probably to me only) function names, but that's ok.

Reply:
Ah, I forgot that we don't support drop yet. Adding support would require new functionality to hook up children and parents, which I would rather do as a separate PR. I'll sprinkle in some `unreachable`s instead to try to achieve the same thing.